routes/v1/auth/controller.auth.js

```diff
@@ -0,0 +1,53 @@
+const { OAuth2Client } = require('google-auth-library');
+const authService = require('./service.auth');
+
+
+const client = new OAuth2Client(process.env.GOOGLE_CLIENT_ID);
+
+
+// handles google user authentication using token provided from frontend
+// return a authToken to be used for logged in users
+const handleAuthenticationRequest = async (req, res) => {
```

Comment:
function name should be like handleGoogleAuthenticationRequest or handleGoogleAuth...